Dockerfile

```diff
@@ -0,0 +1,60 @@
+FROM registry.fedoraproject.org/fedora:26
+
+ENV PYCURL_SSL_LIBRARY=nss
+ENV CFME_ENV=/
+
+RUN dnf install -y --setopt=tsflags=nodocs \
+    git \
+    fluxbox \
+    openssh-server \
+    python \
+    tigervnc-server \
+    freetype-devel \
+    gcc \
+    gcc-c++ \
+    libcurl-devel \
+    libffi-devel \
+    libxml2-devel \
+    libxslt-devel \
+    openssl-devel \
+    postgresql-devel \
+    python2-devel \
+    python2-virtualenv \
+    redhat-rpm-config \
+    tesseract \
+    zeromq-devel \
+    && dnf clean all
+
+# Build and setup selenium
+RUN mkdir /selenium; curl -L http://goo.gl/yLJLZg > /selenium/selenium_latest.jar
+ADD dockerfiles/cfme_tests_img/xstartup /root/.vnc/xstartup
+EXPOSE 5999
+
+RUN virtualenv /cfme_venv
+RUN echo "source /cfme_venv/bin/activate" >> /root/.bashrc
+
+#RUN git clone https://github.com/ManageIQ/integration_tests.git
+RUN git clone https://github.com/psav/cfme_tests.git --branch dev_trial integration_tests
+WORKDIR integration_tests
+ADD docker-assets/entrypoint /entrypoint
+
+RUN touch .yaml_key
+RUN mkdir -p ../cfme-qe-yamls/complete/
+
+### DELETE ME ###
+# Temporary manual run of the quickstart, so these can build separate layers
+# in the image, and not require rebuilding one giant layer
+RUN ../cfme_venv/bin/pip install -U pip wheel setuptools_scm docutils
+RUN ../cfme_venv/bin/pip install -r requirements/frozen.txt --no-binary pycurl
```

Comment:
the abi issues with numpy/matplotlib are solved by adding `--no-binary numpy --no-binary matplotlib`

Reply:
Thank you!